python/ray/includes/ray_config.pxi

```diff
@@ -80,6 +81,10 @@ cdef class Config:
     def kill_worker_timeout_milliseconds():
         return RayConfig.instance().kill_worker_timeout_milliseconds()
 
+    @staticmethod
+    def worker_register_timeout_seconds():
```

Comment:
Did you add it for debugging purpose?

Reply:
Ah, no, this not for debugging. We want to expose it so our application can configure it with a different value (> 30s). We ran into a timeout issue with the default 30 seconds, and our training job just got hang. For more context. please refer to the issue I linked with this PR.

Reply:
And it works for me just doing this:
```
ray.init(
    _internal_config=json.dumps({
        "worker_register_timeout_seconds": 120,
    }),
)
```

Reply:
I see. That makes sense